plugins/inputs/opentelemetry/opentelemetry_test.go

```diff
@@ -37,7 +37,6 @@ func TestOpenTelemetry(t *testing.T) {
 	metricExporter, err := otlpmetricgrpc.New(ctx,
 		otlpmetricgrpc.WithInsecure(),
 		otlpmetricgrpc.WithDialOption(
-			grpc.WithBlock(),
```

Comment:
Are you sure to remove this? This will make `connect` to wait until the connection is established IIRC...

Reply:
Hmm, the reason I removed it was it was deprecated and stated not supported by gRPC's NewClient:

```
plugins/inputs/opentelemetry/opentelemetry_test.go:40:4                     staticcheck  SA1019: grpc.WithBlock is deprecated: this DialOption is not supported by NewClient. Will be supported throughout 1.x. 
```

As this was a test case and it ran locally, I had hoped this was ok.

Reply:
Ok, didn't know it was deprecated. :-) Fine with me then...